server/routes/api/protein.py

```diff
@@ -25,10 +30,293 @@
 @bp.route('/<path:dcid>')
 def get_node(dcid):
     """Returns data given a protein node."""
-    response = dc_service.fetch_data('/internal/bio', {
+    response = dc.fetch_data('/internal/bio', {
         'dcid': dcid,
     },
-                                     compress=False,
-                                     post=False,
-                                     has_payload=False)
+                             compress=False,
+                             post=False,
+                             has_payload=False)
     return response
+
+
+def id(id_or_dcid):
```

Comment:
if this function will only be called from within protein.py then consider naming it with an underscore, e.g. _id(). If you expect this function to be called from outside this file, then the function-level comments should follow the Python style guide. See the example in this section: https://google.github.io/styleguide/pyguide.html#383-functions-and-methods

Reply:
Thank you so much for the review! Got it will do!